staff/docker-compose.e2e.yml

```
@@ -34,7 +34,7 @@ services:
     build:
       target: dev
       context: ../participant
-    command: ["npm", "run", "dev:dbreset"]
+    command: [ "npm", "run", "dev:dbreset" ]
     depends_on:
       database-e2e:
```

This is breaking the staff portal tests

```suggestion
      database:
```